_resources/templates/root.html

```diff
@@ -11,6 +11,8 @@
 	<link rel="stylesheet" type="text/css" href="{{asset "document.css"}}" />
 	<link rel="stylesheet" type="text/css" href="{{asset "search.css"}}" />
 	<link rel="stylesheet" type="text/css" href="{{asset "content.css"}}" />
+	<link rel="stylesheet" type="text/css" href="https://cdn.skypack.dev/tippy.js/dist/tippy.css" />
```

Comment:
If you do add a major version to the URLs in the import lines, don't forget to add it here too (and probably add a comment that the two need to be kept in sync).